### PR TITLE
Typography Buttons Colors - starting mantine

### PR DESCRIPTION
**Pull request by ainouzgali**

### CODE REVIEW

File: apps/web/src/design-system/typography/title/Title.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { TitleOrder, Title as MantineTitle } from '@mantine/core';
+
+interface ITitleProps extends JSX.ElementChildrenAttribute {
+  order?: TitleOrder;
```
Comment (scopsy) [inline apps/web/src/design-system/typography/title/Title.tsx:4]: @ainouzgali what do you think about mantine `order` prop? I find it a bit confusing, maybe something like `size` is better here?

**Review by scopsy — COMMENTED** [review]
Hurray for your first big PR! 🎊
Following up our video review, adding a couple of comments for things we discussed during our talk :)

File: apps/web/src/design-system/button/Button.styles.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const getFilledStyles = (theme) => ({
+  backgroundImage: theme.colorScheme === 'dark' ? getGradient(theme, 'dark', 4) : getGradient(theme, 'gray', 2),
+});
+
+const getOutlineStyles = (theme, dark, disabled) => {
+  const disabledColor = dark ? getGradient(theme, 'dark', 6) : getGradient(theme, 'gray', 5);
+  return {
+    border: '1px solid transparent',
+    backgroundImage: `${dark ? getGradient(theme, 'dark', 7) : theme.fn.linearGradient(0, theme.white, theme.white)},${
+      !disabled ? theme.fn.themeColor('gradient', 8) : disabledColor
```
[the remark this answers is not in the document]
Comment (scopsy) [reply apps/web/src/design-system/button/Button.styles.ts:20]: Now I see that it's not a ternary at all 😅  Maybe we can extract it to a variable for [better readability ](https://refactoring.guru/extract-variable)

File: apps/web/src/design-system/typography/title/Title.tsx (hunk 1 above)
Comment (ainouzgali) [reply apps/web/src/design-system/typography/title/Title.tsx:4]: Yes, I've been thinking the same thing. 
Should we call it size with values 'md' and 'lg' to be consistent with the other components?

Comment (scopsy) [reply apps/web/src/design-system/typography/title/Title.tsx:4]: I think it's better use 1,2,3,4,5 (now only 1,2) for a better semantic and mental model resemblance with h1, h2, h3 and etc...

File: apps/web/src/design-system/button/Button.styles.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ainouzgali) [reply apps/web/src/design-system/button/Button.styles.ts:20]: Definitely needed. Will do.

File: apps/web/.storybook/preview.jsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
-import React, { useState } from 'react';
-import { MantineProvider, ColorSchemeProvider, Affix, ActionIcon } from '@mantine/core';
+import React from 'react';
+import { ColorSchemeProvider } from '@mantine/core';
 import { ThemeProvider } from '../src/design-system/ThemeProvider';
 import { useDarkMode } from 'storybook-dark-mode';
 
 export const parameters = {
   layout: 'fullscreen',
+  viewMode: 'docs',
```
Comment (scopsy) [inline apps/web/.storybook/preview.jsx:8]: Cool! This will show it by default?

File: apps/web/src/design-system/config/colors.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export const colors = {
```
Comment (scopsy) [inline apps/web/src/design-system/config/colors.ts:1]: Looks amazing! 🤩

File: apps/web/src/design-system/config/theme.config.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import { MantineThemeOverride } from '@mantine/core';
+import { colors } from './colors';
+
+export const mantineConfig: MantineThemeOverride = {
```
Comment (scopsy) [inline apps/web/src/design-system/config/theme.config.ts:4]: Great you found the MantineThemeOverride interface this will help us a lot

**Review by scopsy — APPROVED** [review]
Fantastic! Let's Ship it! 😎

**Pull request merged**